packages/calcite-components/src/components/action-group/action-group.tsx

```diff
@@ -95,6 +95,16 @@ export class ActionGroup
    */
   @Prop({ reflect: true }) scale: Scale;
 
+  /**
+   * Specifies the component's fallback menu `placement` when it's initial or specified `placement` has insufficient space available.
+   */
+  @Prop() menuFlipPlacements: FlipPlacement[];
+
+  /**
+   * Determines where the action menu will be positioned.
+   */
+  @Prop({ reflect: true }) menuPlacement: LogicalPlacement;
```

Comment:
Do we prefer `actionMenuPlacement` or is `menuPlacement` sufficient?

Reply:
I think the current name is clear and concise.